Source/WebCore/rendering/GridTrackSizingAlgorithm.h

```diff
@@ -218,6 +218,9 @@ class GridTrackSizingAlgorithm final {
     void computeFlexSizedTracksGrowth(double flexFraction, Vector<LayoutUnit>& increments, LayoutUnit& totalGrowth) const;
     double findFrUnitSize(const GridSpan& tracksSpan, LayoutUnit leftOverSpace) const;
 
+
+    void computeIndefiniteItemsForMasonry(MasonryIndefiniteItems&) const;
```

Comment:
I would add a comment here mentioning these are helper functions for Masonry.

Reply:
I think the names help make that clear but I wouldn't mind reorganizing the declarations in the future so that all masonry related things are together!